src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -75,19 +80,22 @@ public String toString() {
     public static class DescriptorImpl extends Descriptor<UserRemoteConfig> {
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project,
```

Comment:
existing method. I would rather see a PR that adds restricted only rather than co-mingling